boilerplates/fastify/files/fastify-entry.ts

```diff
@@ -72,6 +77,15 @@ async function startServer() {
   app.addContentTypeParser("*", function (_request, _payload, done) {
     done(null, "");
   });
+  if (BATI.has("ts-rest")) {
+    app.addContentTypeParser("application/json", { parseAs: "string" }, function (_request, payload, done) {
+      if (typeof payload === "string") {
+        const json = JSON.parse(payload);
+        done(null, json);
+      }
+      done(null, "");
+    });
+  }
```

Comment:
Can be removed once generic handler is used.

Also, never add global `ContentTypeParser` on it, as it will probably clash with generic handlers. If a tool seem to require it, we should probably write a generic handler instead.

Reply:
Noted :+1: